app/views/templates/search/search_object.rb

```diff
@@ -151,8 +155,19 @@ def content
                 }
               ]
             }),
-
-            description: render_document_show_field_value(document, 'proxies.dcDescription'),
+            description: data_section( {
+               :title => 'site.object.meta-label.description',
+               :sections  => [
+                 {
+                   :title  => false,
+                   :collected => render_document_show_field_value(document, 'proxies.dcDescription')
+                 },
+                 {
+                   :title   => false,
+                   :collected  => render_document_show_field_value(document, 'proxies.dctermsTOC')
+                 }
+               ]
+             }),
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.